receiver/vcenterreceiver/metadata.yaml

```diff
@@ -286,7 +294,7 @@ metrics:
       monotonic: false
       value_type: int
       aggregation_temporality: cumulative
-    attributes: []
+    attributes: [memory_usage_type]
```

Comment:
I'm actually thinking that we'll probably need a feature gate (default off for now) for this metric. I believe that's how we'd want to control the additional functionality of the new attribute here? I could be wrong...